tests/setuponlytests/quilt/verify.sh

```diff
@@ -0,0 +1 @@
+mc-image-helper assert fileExists "/data/quilt-server-launch-*.jar"
```

Comment:
Looks like it should be
```suggestion
mc-image-helper assert fileExists "/data/quilt-server-*-launch.jar"
```

Reply:
should be fixed now